src/api/config/routes/webui_routes.rb

```diff
@@ -52,9 +52,11 @@
     resources :package, only: [:index], controller: 'webui/package', constraints: cons
 
     controller 'webui/package' do
-      get 'package/edit/:project/:package' => :edit, constraints: cons, as: 'edit_package'
-      patch 'package/update' => :update, constraints: cons
-      get 'package/show/:project/:package' => :show, as: 'package_show', constraints: cons
+      defaults format: 'js' do
+        get 'package/edit/:project/:package' => :edit, constraints: cons, as: 'edit_package'
+        patch 'package/update' => :update, constraints: cons
+      end
+      get 'package/show/:project/:package' => :show, as: 'package_show', constraints: cons, format: 'html'
```

Comment:
Could this go in the block on the line 63?

Reply:
Yes it can. Let me change it

Reply:
Done